### PR TITLE
SME and QE pull request for BXMSDOC-2941 Getting started with case management

### PR DESCRIPTION
**Pull request by michelehaglund**
Original Jira: https://issues.jboss.org/browse/BXMSDOC-3323
Rendered output: http://file.rdu.redhat.com/~mhaglund/BXMSDOC-3323/

### CODE REVIEW

File: doc-content/enterprise-only/cases/case-management-cancel-or-destroy-case-proc.adoc [hunk 1]
```diff
@@ -1,13 +1,15 @@
 [id='case-management-cancel-or-destroy-case-proc-{context}']
 = Canceling or destroying a case
 
-Cases can be canceled if they are no longer required and do not require any case work to be performed. Cases that are canceled can be reopened later with the same case instance ID and case file data. In some cases, you might want to permanently destroy a case so that it cannot be reopened. 
+*<MH>Cristiano, can this be done using the showcase app?</MH>*
```
Comment (cristianonicolai) [inline doc-content/enterprise-only/cases/case-management-cancel-or-destroy-case-proc.adoc]: It is. Same as the instructions for Close. The Cancel button is now the one hidden in the three dots button for both Case List and Case details.

File: doc-content/enterprise-only/cases/case-management-security-configuration-proc.adoc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [id='case-management-security-configuration-proc-{context}']
 = Configuring security for case management
```
[the remark this answers is not in the document]
Comment (mswiderski) [reply doc-content/enterprise-only/cases/case-management-security-configuration-proc.adoc]: agreed

File: doc-content/enterprise-only/cases/case-management-security-configuration-proc.adoc [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
-
+*<MH>Cristiano, Can this process be simplified by using the showcase app? If so, how? Also, would this configuration be done by the developer, case worker, or process admin?</MH>*
```
Comment (mswiderski) [inline doc-content/enterprise-only/cases/case-management-security-configuration-proc.adoc]: this is done by developer

File: doc-content/enterprise-only/cases/case-management-stage-activation-and-completion-conditions-proc.adoc [hunk 4]
```diff
@@ -1,29 +1,22 @@
 [id='case-management-stage-activation-and-completion-conditions-proc']
 = Configuring stage activation and completion conditions
 
+*<MH>Cristiano, This section seems random to me. Can I remove it for the sake of the tutorial? If it applies to the showcase app, I can add it to another topic.</MH>*
```
[the remark this answers is not in the document]
Comment (mswiderski) [reply doc-content/enterprise-only/cases/case-management-stage-activation-and-completion-conditions-proc.adoc]: +1 to remote this

File: doc-content/enterprise-only/cases/case-management-designing-a-case-proc.adoc [hunk 5]
```diff
@@ -1,120 +1,257 @@
...
+. Expand *Milestone* in the Object Library and drag a new milestone on the canvas and place it on the right side of the *Place order* end event.
+. Click on the new milestone and in the upper-right corner, click image:cases/left-arrows.png[] to open the *Properties* panel.
+. Input `Hardware spec ready` in the *Name* field.
+. Select `true` from the *Adhoc autostart* menu.
+. Click the down arrow in the *Assignments* field and add the following new conditions in the *Hardware spec ready Data I/O* window.
 +
-* `owner`: The employee who is making the hardware order request. There can be only one person assigned to this role.
-* `manager`: The employee's manager; the person who will approve or deny the requested hardware. The role cardinality is set to `1`, which means that only one person or group can be assigned to this role.
-* `supplier`: The available suppliers of IT hardware in the system. There is usually more than one supplier.
+image::cases/hardware-io.png[Hardware spec ready I/O values]
```
[the remark this answers is not in the document]
Comment (mswiderski) [reply doc-content/enterprise-only/cases/case-management-designing-a-case-proc.adoc]: in 7.2 you can use shorten version for the expression

`CaseData(data.get("hwSpec") != null)`

File: doc-content/enterprise-only/cases/case-management-it-order-sample-proc.adoc [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+.Procedure
+
+Navigate to the *IT_Orders* sample project to view the predefined assets.
+
+. Log in to {CENTRAL} and click *Menu* -> *Design* -> *Projects*.
+. Click image:project-data/dots.png[] in the upper-right corner of the screen and select *Try Samples*.
+. Select *IT_Orders* and click *Ok*.
+
+The *Assets* view of the project opens. Click on each asset to get gain a general understanding of what you will be creating in your new project. You can refer back to the *IT_Orders* sample project's assets for reference as needed.
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-it-order-sample-proc.adoc]: Done

File: doc-content/enterprise-only/cases/case-management-deploy-test-proc.adoc [hunk 7]
```diff
@@ -0,0 +1,18 @@
+[id='case-management-deploy-test-proc']
+= Deploying and testing the IT order case project
+
+*<MH>Cristiano and Juraj, can you please provide me with the steps to deploy and test the outcome?</MH>*
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-deploy-test-proc.adoc]: Done. You can check the latest draft at:  http://file.rdu.redhat.com/~mhaglund/BXMSDOC-3323/

File: assemblies/assembly_case-management-getting-started/main.adoc [hunk 8]
```diff
@@ -11,30 +11,42 @@ include::_artifacts/author-group.adoc[]
...
 
-Case management differs from business process management (BPM) in that it focuses on the actual data being handled throughout the case and less on the sequence of steps taken to complete a goal. Case data is the most important piece of information in case handling, while business context and decision-making is in the hands of the human case worker.
-
-Use this document to get started with case management while referring to the IT Orders sample project in {CENTRAL}. Deploy the project in {CENTRAL} to view and interact with the case assets described in this tutorial, or recreate the project yourself.
+This document will familiarize you with the concepts of case management by showing you how to create and deploy a new IT order project. Before you create the new project, you will review the existing *IT_Orders* sample project in {CENTRAL} for reference. Additionally, you will use the Case management Showcase application, which is included in the {PRODUCT} to demonstrate the capabilities of case management.
 
+*<MH>Cristiano, we need to complete the process by adding steps for deploying the project and how and where they review the results</MH>*
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply assemblies/assembly_case-management-getting-started/main.adoc]: Got it. Thanks :)

File: assemblies/assembly_case-management-getting-started/main.adoc [hunk 9]
```diff
@@ -11,30 +11,42 @@ include::_artifacts/author-group.adoc[]
...
 
 .Prerequisites
 
-* {EAP_LONG} 7.1.0 is installed. For installation information, see the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.1/html-single/installation_guide/[_Red Hat JBoss EAP 7.1.0 Installation Guide_].
+* {EAP_LONG} 7.2.0 is installed. For installation information, see the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html-single/installation_guide/[_Red Hat JBoss EAP 7.2.0 Installation Guide_].
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply assemblies/assembly_case-management-getting-started/main.adoc]: The URLs won't work until we publish the docs.

File: doc-content/enterprise-only/cases/case-management-cancel-or-destroy-case-proc.adoc (hunk 1 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-cancel-or-destroy-case-proc.adoc]: Got it. Thanks

File: doc-content/enterprise-only/cases/case-management-comments-con.adoc [hunk 10]
```diff
@@ -1,5 +1,7 @@
 = Comments
 
+*<MH>Cristiano, This section seems random to me. Can I remove it for the sake of the tutorial? If it applies to the showcase app, I can add it to another topic.</MH>*
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-comments-con.adoc]: Moved up in the guide below the Case file concept topic.

File: doc-content/enterprise-only/cases/case-management-create-sub-proc-proc.adoc [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+. Click on the start event and create an outgoing connection from the start event to a task and convert the new task to a user task.
++
+image::cases/user-task.png[]
+. Click the user task and in the *Diagram Properties* panel, input `Place order` in the *Name* field.
+. Create an outgoing connection from the *Place order* user task and connect it to an end event.
++
+image::cases/create-end-event.png[]
++
+image::cases/sub-proc.png[]
+. In the diagram editor, click *Save*, then *Save*, to confirm your changes.
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-create-sub-proc-proc.adoc]: Good catch. I have added the missing info.

File: doc-content/enterprise-only/cases/case-management-creating-a-case-proc.adoc [hunk 12]
```diff
@@ -1,23 +1,23 @@
...
-
-. To check that the case project has been set up correctly, click *Settings* -> *Deployments*. 
-. Verify that the *Runtime Strategy* is set to `Per case` and that the *Marshalling Strategies* include a case file marshaller and a documents marshaller.
-
+*Tip*: {CENTRAL} provides a default space called *MySpace*, as shown in the following image. You can use the default space to create and test example projects.
++
 
+.Default spaces
+image::getting-started/spaces-teams-bc.png[]
+. Click *Add Project* -> *Case project*.
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-creating-a-case-proc.adoc]: Added new image.

File: doc-content/enterprise-only/cases/case-management-designing-a-case-proc.adoc [hunk 13]
```diff
@@ -1,120 +1,257 @@
...
-* `name`: The name of the case. This is usually the subject of the case or project that is being case managed.
-* `package`: The location in which case file is to be created.
-* `Case ID prefix (optional)`: A configurable prefix that enables you to easily distinguish different types of cases. The prefix is followed by a generated ID in the format `ID-XXXXXXXXXX`, where `XXXXXXXXXX` is a generated number that provides a unique ID for the case instance. If a prefix is not provided, the default prefix is `CASE` and generates instances with the following identifiers: 
+. Click *Ok* to open the process designer.
+. Define process-level and case-level variables.
+.. Click image:cases/expand-proj-explorer.png[] to view the *Project Explorer*.
+.. Click *BUSINESS PROCESSES (LEGACY)* -> *orderhardware*.
+.. Click image:cases/left-arrows.png[] on the right side of the designer to open the *Diagram properties* panel.
+.. Click the down arrow in the *Variable Definitions* field to open the *Editor for Variable Definitions*.
+.. Click *Add Variable*, and input the following values and click *Ok*:
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-designing-a-case-proc.adoc]: New screen capture added.

**Comment by LuboTerifaj** [comment]
@michelehaglund 

Can you please put your comments in separate comment here instead of updating mine?
It will be much clearer.

Thank you.

**Comment by LuboTerifaj** [comment]
Hello @michelehaglund 

Can you please update my comment to initial state and add your new one here? 

It will be much more readable. It is pretty unclear which comments are yours and what are mine at the moment.

Thank you.

**Pull request merged**